libs/island-ui/core/src/lib/Select/Select.tsx

```diff
@@ -125,122 +93,59 @@ export const Select = ({
           Menu,
           Option,
         }}
+        isClearable
+        backspaceRemovesValue
```

Comment:
I think this doesn't currently apply to the default one because it's not clearable. I think this should be done when/if the default one ever becomes clearable.